### PR TITLE
Feat/article meta

### PR DESCRIPTION
**Pull request by thatportugueseguy**
captura de ecra 2017-08-02 as 15 41 16

### CODE REVIEW

File: packages/article-meta/__tests__/article-meta.test.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+/* eslint-env jest */
```
Comment (sericaia) [inline packages/article-meta/__tests__/article-meta.test.js]: you dont need to have __tests__ folder if you only have one test :) this component is quite small, I think you can move the test to the parent one

File: packages/article-meta/article-meta.test.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+/* eslint-env jest */
```
Comment (sericaia) [inline packages/article-meta/article-meta.test.js]: the 4th test! :P replace this file with the other

Comment (thatportugueseguy) [inline packages/article-meta/article-meta.test.js]: done :)

File: packages/article-meta/article-meta.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import React from "react";
```
Comment (craigbilner) [inline packages/article-meta/article-meta.js]: is `article-meta` the "official" name for this?

File: packages/article-meta/article-meta.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+const ArticleMeta = ({ date, publication, style }) => {
+  const DATE_FORMAT = "dddd MMMM DD YYYY";
+
+  if (!date || !publication) {
+    return null;
+  }
+
+  return (
+    <Text style={[styles.meta, style]}>
+      {format(date, DATE_FORMAT)}, {publication}
```
Comment (craigbilner) [inline packages/article-meta/article-meta.js]: I assume we're going to format dates several times like this, it's effectively `renderPublicationDetails` from `article-summary`?

File: packages/article-meta/article-meta.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    </Text>
+  );
+};
+
+ArticleMeta.propTypes = {
+  date: PropTypes.string,
```
Comment (craigbilner) [inline packages/article-meta/article-meta.js]: should dates be dates?

File: packages/article-meta/article-meta.test.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+  const tree = renderer.create(<ArticleMeta />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
+
+it("renders an article-meta component with content", () => {
+  const tree = renderer.create(<ArticleMeta {...props} />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
+
+it("renders an article-meta component with content", () => {
```
Comment (craigbilner) [inline packages/article-meta/article-meta.test.js]: renders an article-meta component with *red* content

File: packages/article-meta/__snapshots__/article-meta.test.js.snap [hunk 7]
```diff
@@ -0,0 +1,49 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`renders an article-meta component 1`] = `null`;
```
Comment (craigbilner) [inline packages/article-meta/__snapshots__/article-meta.test.js.snap]: is this a good test?

File: packages/article-meta/article-meta.js (hunk 3 above)
Comment (thatportugueseguy) [inline packages/article-meta/article-meta.js]: I think so, as it's got the meta information for the article and that's the name for the [jira issue](https://nidigitalsolutions.jira.com/browse/REPLAT-329). Who should i confirm this with?

File: packages/article-meta/article-meta.js (hunk 4 above)
Comment (thatportugueseguy) [inline packages/article-meta/article-meta.js]: Yes, it is. As far as I understood, we're reusing this component later on and in different publications, so we're transforming it in a module of it's own.

I've talked with Daniela and after this PR is merged, i'll create a new one to replace ```renderPublicationDetails``` for this component

File: packages/article-meta/article-meta.js (hunk 5 above)
Comment (thatportugueseguy) [inline packages/article-meta/article-meta.js]: I'm not sure what you mean. Was it wether propType should be of type date? AFAIK there's no propType for dates, so I used string.

Was this what you meant?

File: packages/article-meta/article-meta.test.js (hunk 6 above)
Comment (thatportugueseguy) [inline packages/article-meta/article-meta.test.js]: yes, needs correction

File: packages/article-meta/__snapshots__/article-meta.test.js.snap (hunk 7 above)
Comment (thatportugueseguy) [inline packages/article-meta/__snapshots__/article-meta.test.js.snap]: Well, if we're not passing any properties, we don't want it to render anything. React handles nulls properly and doesn't render them or break.

File: packages/article-meta/article-meta.js (hunk 3 above)
Comment (craigbilner) [inline packages/article-meta/article-meta.js]: I'm checking with UX

**Comment by craigbilner** [comment]
@thatportugueseguy probably not going to get consensus, can we change to `date-publication` or similar?

File: packages/article-meta/article-meta.js (hunk 3 above)
Comment (thatportugueseguy) [inline packages/article-meta/article-meta.js]: Changed to date-publication

File: packages/article-meta/article-meta.js (hunk 5 above)
Comment (thatportugueseguy) [inline packages/article-meta/article-meta.js]: changed. now expects dates to be passed in.

**Comment by trevorah** [comment]
Err... so here is the meta component on web and ios right now:

screen shot 2017-08-07 at 14 05 41

img_0363

@thatportugueseguy is there another format you are basing this off?

**Comment by craigbilner** [comment]
@trevorah 
screen shot 2017-08-07 at 17 49 53

**Pull request merged**